staging/src/k8s.io/apiextensions-apiserver/test/integration/basic_test.go

```diff
@@ -711,6 +728,26 @@ func TestPatch(t *testing.T) {
 	}
 }
 
+func expectInt64(t *testing.T, obj map[string]interface{}, value int64, pth ...string) {
```

Comment:
can we add this in [`helpers.go`](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/helpers.go)? Looks super useful honestly.